quaddtype/numpy_quaddtype/_quaddtype_main.pyi

```diff
@@ -64,8 +64,15 @@ class QuadPrecDType(np.dtype[QuadPrecision]):  # type: ignore[misc, type-var]  #
     @override
     def __getitem__(self, key: Never, /) -> Self: ...  # type: ignore[override]
 
+# NOTE: Until `QuadPrecision` will become a subclass of `np.generic`, this class cannot
```

Comment:
```
In [5]: issubclass(npq.QuadPrecDType().type, np.generic)
Out[5]: True

In [6]: npq.QuadPrecDType().type
Out[6]: numpy_quaddtype.QuadPrecision

In [7]: issubclass(npq.QuadPrecision, np.generic)
Out[7]: True
```

I think it is as Quadprecision already subclassing the np.floating

Reply:
Nice! I'll open a separate PR for that then.